tests/connection/broker.cpp

```diff
@@ -0,0 +1,68 @@
+
+/* Copyright (c) 2014-2015, Human Brain Project
+ *                          Stefan.Eilemann@epfl.ch
+ */
+
+#define BOOST_TEST_MODULE zeq_connection_broker
+
+#include "../broker.h"
+#include <zeq/connection/broker.h>
+
+#include <lunchbox/sleep.h>
+#include <lunchbox/thread.h>
+#include <lunchbox/servus.h>
+#include <boost/bind.hpp>
+
+using boost::lexical_cast;
+
+const unsigned short port = (lunchbox::RNG().get<uint16_t>() % 60000) + 1024;
+bool received = false;
```

Comment:
a global variable, really?
Might want to put this into the Service class w/ dedicated getter and make it atomic since it's accessed by two threads concurrently.